content/templates/pages/ab_testing.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
it seems like this will be a URL change for all of these templates – e.g. `https://developers.cloudflare.com/workers/templates/snippets/ab_testing/` will now be `https://developers.cloudflare.com/workers/templates/pages/ab_testing/` – is this a temporary thing for this PR, or are we moving forward with the URL change? if so, we should prepare a change to the workers script to redirect any broken links that will happen after merging this

Reply:
Yes great point! I was planning on doing this after to not bloat this PR, even more, I created an issue for tracking : https://github.com/cloudflare/workers-docs/issues/535